src/pages/Thanks/ThanksPage.vue

```diff
@@ -473,6 +493,29 @@ export default {
 			fragment: experimentVersionFragment,
 		}) || {};
 		this.enableMayChallengeHeader = shareChallengeExpData?.version === 'b';
+
+		// Only show the old page if the first page is a borrower page
+		if (this.$appConfig?.firstPage?.indexOf('/lend/') !== -1) {
+			const url = this.$appConfig.firstPage?.split('/')[2];
+			const firstVisitloanId = url?.[2] ?? null;
```

Comment:
it might be also good to check that url[1] is 'lend' so that we know that url[2] is a loan id and not something else.